core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -998,6 +998,33 @@ public long getMajorCompactionSize() {
     return compactionSize;
   }
 
+  /**
+   * returns minor compaction size value from carbon properties or 0 if it is not valid or
+   * not configured
+   *
+   * @return compactionSize
+   */
+  public long getMinorCompactionSize() {
+    long compactionSize = -1;
+    try {
+      compactionSize = Long.parseLong(getProperty(
+              CarbonCommonConstants.CARBON_MINOR_COMPACTION_SIZE, "0"));
+    } catch (NumberFormatException e) {
+      LOGGER.warn("The specified value for property "
+              + CarbonCommonConstants.CARBON_MINOR_COMPACTION_SIZE + " is incorrect."
+              + " Correct value should be long value great than 1. Taking the default value -1" +
+              " which means not consider segment size in minor compaction.");
```

Comment:
Please change as below:

`Invalid value is configured for property CarbonCommonConstants.CARBON_MINOR_COMPACTION_SIZE, considering the default value -1 and not considering segment Size during minor compaction.`